motion/ruby_motion_query/rect.rb

```diff
@@ -376,6 +369,15 @@ def rect_hash_to_rect_array(view, existing_rect, params, grid = nil)
         [l,t,w,h]
       end
 
+      def value_from_symbol(size_symbol, max_value)
+        width_height_symbols = {full: 1, half: 0.5, quarter: 0.25}
```

Comment:
This is nice code of course, but I'm guessing an if/elsif/elsif would be faster than creating a hash each time; Performance-wise. Every ms we can squeeze out of this helps on every view in every app :-)